lib/utilities/walk-up-path.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var path = require('path');
+
+function walkUp(thePath) {
```

Comment:
We're including it into ember-cli and people may try and use our utils, we should account for that edge case when it's relatively simple to handle.